src/exec/job.rs

```diff
@@ -39,7 +41,7 @@ pub fn job(
         // Drop the lock so that other threads can read from the receiver.
         drop(lock);
         // Generate a command, execute it and store its exit code.
-        results.push(cmd.execute(&value, Arc::clone(&out_perm), buffer_output))
+        results.push(cmd.execute(&dir_entry, Arc::clone(&out_perm), buffer_output))
```

Comment:
I messed this up when I resolved conflicts. This should be:
```suggestion
        results.push(cmd.execute(dir_entry.path(), Arc::clone(&out_perm), buffer_output))
```